lib/jormungandr/test/migration/migration-test.hs

```diff
@@ -113,6 +115,15 @@ testMain tr serverPort testAction launchArgs = do
             logError tr ("Process exited with status " <> T.pack (show st))
             pure (ExitFailure 14)
 
+-- | Scan through cardano-wallet server CLI arguments to find a port.
+-- Returns the default 8090 if none found.
+getServerPort :: [String] -> Int
+getServerPort = fromMaybe 8090 . portArg
```

Comment:
Hi @rvl

Here the default seems to have changed from `9090` to `8090`.

Is that your intention?

Reply:
It's a bit of a magic assumption that the cardano-wallet server default is 8090 when the `--port` arg it missing (which it isn't). When we port the db migration test to byron I have a plan to fix this properly.